### ISSUE TITLE
When are we getting the update that the X-Max 3 just got not long ago?

### ISSUE THREAD

**Issue by TrollPlayer94** [QIDITECH/QIDI_PLUS3#37]
Because it has a fix for a issue that the X-Plus 3 also has 'Support qidistudio software to display preview images.' and I would love this for my X-Plus 3 also.

**Comment by QIDITECH** [comment]
Yes, we are actively working on an update to X-Plus 3 that will be released soon.

**Comment by Phil1988** [comment]
Meanwhile, you @TrollPlayer94  may want to have a look at [**FreeDi**](https://github.com/Phil1988/FreeDi) ;)
